docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc

```diff
@@ -4239,23 +4239,44 @@ For the searched DELETE form, if no WHERE clause exists, all rows are deleted fr
 ```
 Searched DELETE is:
 
-DELETE FROM table
-
-    [WHERE search-condition ]
-       [[for] access-option access]
+DELETE 
+[WITH NO ROLLBACK] ["["FIRST N"]"]
+FROM table-name
+[WHERE search-condition ]
+[[for] access-option access]
 
 access-option is:
-    read committed
+read committed
 ```
 
 [[delete_syntax]]
 === Syntax Description of DELETE
 
-* `_table_`
+* `_table-name_`
 +
 names the user table or view from which to delete rows. _table_ must be a base table or an updatable view. To refer to a
 table or view, use the ANSI logical name. See <<database_object_names,Database Object Names>>.
 
+* `WITHOUT NO ROLLBACK`
++
+performs non-transactional delete.

```

Comment:
You may want to explain what a non-transactional delete is (unless it's explained somewhere else and I missed it). A non-transactional delete is one that happens outside a transaction. That means that if the statement is interrupted, any deletes that have been done are still permanent.

Reply:
Thanks Dave, I've added this explanation :raising_hand: